lang/java/maven-plugin/src/main/java/org/apache/avro/mojo/AbstractAvroMojo.java

```diff
@@ -237,6 +245,15 @@ private void compileFiles(String[] files, File sourceDir, File outDir) throws Mo
     }
   }
 
+  protected SpecificCompiler.DateTimeLogicalTypeType getDateTimeLocalTypeType() {
+    try {
+      String upper = String.valueOf(this.dateTimeLogicalTypeType).trim().toUpperCase();
+      return SpecificCompiler.DateTimeLogicalTypeType.valueOf(upper);
+    } catch (IllegalArgumentException e) {
+      return SpecificCompiler.DateTimeLogicalTypeType.JODA;
```

Comment:
Not sure if it's wise to quietly assume any unrecognized value means `JODA`.

Maybe only a `null` or empty string should be interpreted to mean the default of `JODA`? Or maybe just log a warning message?